docs/sdks/languages/rust.mdx

```diff
@@ -2,5 +2,494 @@
 title: "Infisical Rust SDK"
 sidebarTitle: "Rust"
 icon: "/images/sdks/languages/rust.svg"
-url: "https://github.com/Infisical/rust-sdk?tab=readme-ov-file#infisical--the-official-infisical-rust-sdk"
----
\ No newline at end of file
+---
+
+The Infisical Rust SDK ([docs.rs](https://docs.rs/infisical)) provides a convenient and ergonomic way to interact with Infisical programmatically using modern and idiomatic Rust.
```

Comment:
Nit: I would follow one convention for all the SDK docs at the top. We should just say something like "If you’re working with __, the official Infisical __ SDK package is the easiest way to fetch and work with secrets for your application."